### PR TITLE
chore/replacing jcenter

### PR DESCRIPTION
**Pull request by ogustavobelociandt**
Considering jcenter will shut down soon, it is suggested to replace jcenter repository for mavenCentral.
Flutter packages are already migrated to it > https://github.com/flutter/flutter/issues/82847

### CODE REVIEW

File: flutter_google_places_sdk_android/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.1.3
+
+* replacing jcenter for mavinCentral
```
Comment (matanshukry) [inline flutter_google_places_sdk_android/CHANGELOG.md:3]: mavinCentral / mavenCentral

File: flutter_google_places_sdk_android/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 0.1.3
```
Comment (matanshukry) [inline flutter_google_places_sdk_android/CHANGELOG.md:1]: This change isn't that big - use 0.1.2+3

File: flutter_google_places_sdk_android/pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: flutter_google_places_sdk_android
 description: A Flutter plugin for google places sdk that uses the native libraries on each platform
-version: 0.1.2+2
+version: 0.1.3
```
Comment (matanshukry) [inline flutter_google_places_sdk_android/pubspec.yaml:3]: 0.1.2+3

**Review by matanshukry — CHANGES_REQUESTED** [review]

File: flutter_google_places_sdk_android/CHANGELOG.md (hunk 1 above)
Comment (ogustavobelociandt) [reply flutter_google_places_sdk_android/CHANGELOG.md]: solved > https://github.com/matanshukry/flutter_google_places_sdk/pull/10/commits/7809c0f45fad710dd292e34d50f26084799e1691

File: flutter_google_places_sdk_android/CHANGELOG.md (hunk 2 above)
Comment (ogustavobelociandt) [reply flutter_google_places_sdk_android/CHANGELOG.md]: solved > https://github.com/matanshukry/flutter_google_places_sdk/pull/10/commits/7809c0f45fad710dd292e34d50f26084799e1691

**Pull request merged**